evm/utils/fixture_tests.py

```diff
@@ -284,7 +284,7 @@ def normalize_transactiontest_fixture(fixture):
     try:
         normalized_fixture['rlp'] = decode_hex(fixture['rlp'])
     except binascii.Error:
-        normalized_fixture['rlpHex'] = fixture['rlp']
```

Comment:
@pipermerriam I changed this back and am now checking for for the `KeyError` when the tests try to call `fixture['rlp']` (when it's now `fixture['rlpHex']`)